web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.utils.tsx

```diff
@@ -0,0 +1,151 @@
+  return collapsed ? i18n._(msg`${shortHeading}`) : i18n._(msg`${fullHeading}`);
+};
+
+// Helper to wrap icons consistently
+const wrapIcon = (Icon: React.FC<any>, props: any = {}) => (
+  <span className="text-[24px] leading-none">
+    <Icon fontSize="inherit" linearGradient {...props} />
+  </span>
+);
+
+// Get credits section
+const getCreditsSection = (loginDisabled: boolean, collapsed: boolean): DashboardNavigationSection => ({
+  heading: getResponsiveHeading('Manage credits', 'Credits', collapsed),
+  items: [
+    {
+      label: i18n._(msg`Portfolio`),
+      icon: wrapIcon(CreditsIcon, { disabled: loginDisabled }),
+      path: 'portfolio',
+    },
+    {
+      label: i18n._(msg`Sell`),
+      icon: <ShoppingCartIcon linearGradient disabled={loginDisabled} />,
+      path: 'sell',
+      disabled: loginDisabled,
+      disabledTooltipText: i18n._(NOT_SUPPORTED_TOOLTIP_TEXT),
+    },
+  ],
+});
+
+// Get projects section
+const getProjectsSection = (loginDisabled: boolean, collapsed: boolean): DashboardNavigationSection => ({
+  heading: getResponsiveHeading('Manage projects', 'Projects', collapsed),
+  items: [
+    {
+      label: i18n._(msg`Projects`),
+      icon: <ProjectsIcon linearGradient disabled={loginDisabled} />,
+      path: 'projects',
+      disabled: loginDisabled,
+      disabledTooltipText: i18n._(NOT_SUPPORTED_TOOLTIP_TEXT),
+    },
+  ],
+});
+
+// Get user-specific sections
+const getUserSections = (loginDisabled: boolean): DashboardNavigationSection[] => [
+  {
+    heading: i18n._(msg`Orders`),
+    items: [
+      {
+        label: i18n._(msg`My orders`),
+        icon: <ShoppingBagIcon linearGradient />,
+        path: 'orders',
+      },
+    ],
+  },
+  {
+    heading: i18n._(msg`Profile`),
+    items: [
+      {
+        label: i18n._(msg`Edit personal profile`),
+        icon: <UserMenuIcon linearGradient disabled={loginDisabled} />,
+        path: 'profile',
+        disabled: loginDisabled,
+        disabledTooltipText: i18n._(NOT_SUPPORTED_TOOLTIP_TEXT),
+      },
+      {
+        label: i18n._(msg`Settings`),
+        icon: <CogIcon linearGradient disabled={loginDisabled} />,
+        path: 'settings',
+        disabled: loginDisabled,
+        disabledTooltipText: i18n._(NOT_SUPPORTED_TOOLTIP_TEXT),
+      },
+    ],
+  },
+];
+
+// Get org-specific sections
+const getOrgSection = (collapsed: boolean): DashboardNavigationSection => ({
+  heading: getResponsiveHeading('Manage organization', 'Org', collapsed),
+  items: [
+    {
+      label: i18n._(msg`Edit org profile`),
+      icon: <OrgProfile linearGradient />,
+      path: 'profile',
+    },
+    {
+      label: i18n._(msg`Members`),
+      icon: <MembersIcon linearGradient />,
+      path: 'members',
+    },
+  ],
+});
+
+// Get logout section (always the same)
+const getLogoutSection = (): DashboardNavigationSection => ({
+  heading: '',
+  items: [
+    {
+      label: i18n._(msg`Log out`),
```

Comment:
"log out" text should be grey https://www.figma.com/design/L3VpDxt1PXTATPbSY0Z7DV/Organizations--org-member-permissions--and-project-collaborator-permissions?node-id=7082-122179&m=dev
this corresponds to `text-sc-text-sub-header` tailwind class